.github/workflows/dispense_volume.yml

```diff
@@ -77,6 +77,7 @@ jobs:
           echo "MAINNET_RPC_URL=https://mainnet.infura.io/v3/" >> $GITHUB_ENV
           echo "SAPPHIRE_MAINNET_RPC_URL=https://sapphire.oasis.io/" >> $GITHUB_ENV
           echo "GOERLI_RPC_URL=https://rpc.ankr.com/eth_goerli" >> $GITHUB_ENV
+          echo "ENERGYWEB_RPC_URL=https://rpc.energyweb.org" >> $GITHUB_ENV
```

Comment:
Ideally this change is in a different PR & different issue. Easier to track what's going on. Same with the other energy-web change.